object_manager.go

```diff
@@ -58,6 +58,8 @@ type IBObjectManager interface {
 	GetNetworkViewByRef(ref string) (*NetworkView, error)
 	GetPTRRecord(dnsview string, ptrdname string, recordName string, ipAddr string) (*RecordPTR, error)
 	GetPTRRecordByRef(ref string) (*RecordPTR, error)
+	GetTXTRecord(name string) (*RecordTXT, error)
```

Comment:
What about searching in a DNS view with a name other than 'default'?

Reply:
dnsView input added similar to GetARecord